src/CheckNumBox.cpp

```diff
@@ -122,6 +122,22 @@ CheckNumBox::CheckNumBox(const BRect &frame, const char *name, const char *label
 
 	SetCharacterLimit(8);
 }
+CheckNumBox::CheckNumBox(const char *name, const char *label,
```

Comment:
You can get rid of the non-layout constructors on this and all the other custom classes.